patches/minecraft/net/minecraft/client/gui/inventory/GuiContainer.java.patch

```diff
@@ -1,14 +1,74 @@
 --- ../src-base/minecraft/net/minecraft/client/gui/inventory/GuiContainer.java
 +++ ../src-work/minecraft/net/minecraft/client/gui/inventory/GuiContainer.java
-@@ -116,6 +116,7 @@
+@@ -1,15 +1,14 @@
+ package net.minecraft.client.gui.inventory;
+ 
+-import com.google.common.collect.Sets;
+ import java.io.IOException;
+ import java.util.Set;
++
+ import net.minecraft.client.Minecraft;
+ import net.minecraft.client.gui.GuiScreen;
+ import net.minecraft.client.renderer.GlStateManager;
+ import net.minecraft.client.renderer.OpenGlHelper;
+ import net.minecraft.client.renderer.RenderHelper;
+ import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+-import net.minecraft.client.renderer.texture.TextureMap;
```

Comment:
Don't change imports.

Reply:
uhm `import net.minecraft.client.renderer.texture.TextureMap;` appears as not used import